js/client/modules/@arangodb/testing.js

```diff
@@ -82,6 +82,7 @@ let optionsDocumentation = [
   '   - `prefix`:    prefix for the tests in the xml reports',
   '',
   '   - `rr`: if set to true arangod instances are run with rr',
+  '   - `coreCheck`: if set to true we will try to locate coredumps and get a backtrace from it',
```

Comment:
"if set to true, we will attempt to locate a coredump to produce a backtrace in the event of a crash"